products/ocp4/profiles/stig.profile

```diff
@@ -0,0 +1,18 @@
+documentation_complete: true
+
+platform: ocp4
+
+metadata:
+    SMEs:
+        - jhrozek
```

Comment:
Can you also add the following GH nicks to the SME list:
 - `@Vincent056`
 - `@mrogers950`
 - `@rhmdnd`
 - `@david-rh`